plans/destroy.pp

```diff
@@ -1,9 +1,9 @@
 plan autope::destroy(
-  TargetSpec              $targets          = get_targets('peadm_nodes'),
-  Enum['google', 'aws']   $provider         = 'google',
-  String[1]               $project          = $provider ? { 'aws' => 'ape', default => 'oppenheimer' },
-  String[1]               $ssh_user         = $provider ? { 'aws' => 'centos', default => 'oppenheimer' },
-  String[1]               $cloud_region     = $provider ? { 'aws' => 'us-west-2', default => 'us-west1' },
+  TargetSpec                       $targets          = get_targets('peadm_nodes'),
+  Enum['google', 'aws', 'azure']   $provider         = 'google',
+  String[1]                        $project          = $provider ? { 'aws' => 'ape', default => 'oppenheimer' },
+  String[1]                        $ssh_user         = $provider ? { 'aws' => 'centos', default => 'oppenheimer' },
```

Comment:
@davidsandilands Do we want to specify azure defaults?

Reply:
I marked the location since it was different from aws and google but didn't think there was a need for the others since the access account is created to whatever name is provided and the project name is just a tag/name for resource group and tags with no logical difference for azure